contracts/Splitter.sol

```diff
@@ -44,22 +48,24 @@ contract Splitter is Pausable {
     }
 
     function withdraw(uint amount) public whenNotPaused returns(bool) {
-        uint256 msgSender = balances[msg.sender];
+        uint256 senderBalance = balances[msg.sender];
```

Comment:
More like the balance of the withdrawer / requester, no? The contract is the sender here.

Reply:
I see what you mean.